### ISSUE TITLE
can't find <wrong_path>/fusion/main.py

### ISSUE THREAD

**Issue by datakid** [Oshlack/Clinker#2]
When running - as recommended - the command in this format:

`bpipe -p out="/path/to/clinker/output" -p caller="/path/to/jaffa_results.csv" -p col="3,4,5,6" -p del="c" -p genome="19" -p print="true" -p fusions="BCR:ABL1" -p pdf_width="9" -p pdf_height="16" -p sizing="1,3,1,2,4,2" -p competitive="false" /path/to/clinker/clinker.pipe /path/to/*.fastq.gz`

from /home/<user>/<my_clinker_project> I get the error:

[root@expanded01 tmp_liz]# bpipe errors

============================== Found 1 failed commands from run 6944 ===============================

============================================ Command 1 =============================================

Command    : python /home/<user>/<my_clinker_project>/fusion/main.py -in /path/to/jaffa/jaffa_results.csv -out /path/to/clinker_test_output -pos 3,4,5,6 -del c -genome 19 -competitive false

Exit Code  : 2
Output     : 

        python: can't open file '/home/<user>/<my_clinker_project>/fusion/main.py': [Errno 2] No such file or directory

So this is a problem in a shared environment, as I wont put clinker directories in everyone's directory and I wont have people changing into /bin folders to get this right.

Can we set that up so that it is relative or uses a var

**Comment by breons** [comment]
Hi again! That's a shame, I thought it was working after the last issue got resolved?

You might not have seen my last message in that closed issue, but I made an adjustment (see below). 

> 
Hi again,

I've made an update that will make Clinker easier to install/use in an environment such as yours.

Basically you just set an environment variable called "CLINKERDIR" that directs to the clinker directory (i.e. export CLINKERDIR="/path/to/clinkerdir"), users can just run the below snippet and it should all just click:

bpipe -p out="/path/to/clinker/output" -p caller="/path/to/jaffa_results.csv" -p col="3,4,5,6" -p del="c" -p genome="19" -p print="true" -p fusions="BCR:ABL1" -p pdf_width="9" -p pdf_height="16" -p sizing="1,3,1,2,4,2" -p competitive="false" $CLINKERDIR/workflow/clinker.pipe /path/to/*.fastq.gz

> 

You will notice in clinker.pipe that there is a "fst_program = System.getenv("CLINKERDIR")" line. Feel free to either create a system variable or manually change it to that shared path.

Does that help?
